packages/components/select/src/select.ts

```diff
@@ -8,13 +10,16 @@ import { buildProps, definePropType, iconPropType } from '@element-plus/utils'
 import { useTooltipContentProps } from '@element-plus/components/tooltip'
 import { ArrowDown, CircleClose } from '@element-plus/icons-vue'
 import { tagProps } from '@element-plus/components/tag'
+import { CHANGE_EVENT, UPDATE_MODEL_EVENT } from '@element-plus/constants'
+
 import type {
   Options,
   Placement,
   PopperEffect,
 } from '@element-plus/components/popper'
+import type { OptionValue } from './type'
 
-export const SelectProps = buildProps({
+export const selectProps = buildProps({
```

Comment:
In order not to affect the original users, perhaps we can export both `dada` and `dada` at the same time, and remove the capitalized one in a future version.